packages/shared/src/components/modals/squads/PrivilegedMembersModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement } from 'react';
+import Link from 'next/link';
+import { Modal, ModalProps } from '../common/Modal';
+import { SourceMember } from '../../../graphql/sources';
+import { UserShortInfo } from '../../profile/UserShortInfo';
+import SquadMemberBadge from '../../squads/SquadMemberBadge';
+
+export interface PrivilegedMembersModalProps
+  extends Omit<ModalProps, 'children'> {
+  members: SourceMember[];
+}
+
+function PrivilegedMembersModal({
+  members,
+  ...props
+}: PrivilegedMembersModalProps): ReactElement {
+  return (
+    <Modal kind={Modal.Kind.FlexibleCenter} size={Modal.Size.Medium} {...props}>
+      <Modal.Header title="Moderated by" />
+      <Modal.Body className="!p-0">
```

Comment:
Just checking can't we re-use `UserListModal` is also has search for instance.

Reply:
Tried it at first, but it was an infinite modal. Thought of adding a more generic modal for this new implementation but the object is `SourceMember` which is not very common (but this one can still be reused). I tried the existing SquadMembersModal and add a filter on the prop but it can only accept one Role at a time which we need to show both admin and moderators.

Reply:
Ah ok, this doesn't return infinite query?
Assumed it was a callable like that.

But let's not over complicate for now, just thought it would be nice to have one base.

Reply:
Yeah, totally agree with that. Was really hoping to introduce a generic user list modal. For this one, we are just pulling the data from the `squad.privilegedMembers` object which is not from an infinite query.